picard/ui/options/scripting.py

```diff
@@ -98,6 +99,88 @@ def highlightBlock(self, text):
         self.setCurrentBlockState(open_brackets)
 
 
+class AdvancedScriptItem(QtGui.QWidget):
+    """Custom widget for script list items"""
+
+    _CHECKBOX_POS = 0
+    _BUTTON_UP = 1
+    _BUTTON_DOWN = 2
+    _BUTTON_OTHER = 3
+
+    def __init__(self, name=None, state=True, parent=None):
+        super(AdvancedScriptItem, self).__init__(parent)
+        layout = QtGui.QGridLayout()
+        layout.setHorizontalSpacing(5)
+        layout.setVerticalSpacing(2)
+        layout.setContentsMargins(0, 0, 0, 0)
+        self.setLayout(layout)
+
+        checkbox = QtGui.QCheckBox()
+        checkbox.setChecked(state)
+        checkbox.setText(name)
```

Comment:
This is a bit awkward, because when one wants to select another script by click, clicking on the text check/uncheck the checkbox, instead of selecting the line.
I think you should use a checkbox without text in first column and use another column for script name.